### PR TITLE
Help Page Update 2

### PR DESCRIPTION
**Pull request by antares1470**
### Description of the Change

- removed analytic help pages in the Tools section, tidied up the analytic help pages in the Analytic View section
- added help pages for developer preferences and graph preferences
- removed help page for display preferences
- moved the label fonts help page from Visualisation section to Preferences section
- updated the following help pages:
    - all the analytic questions
    - Find View
    - Table View (Experimental)
    - datetime range global parameter
    - application preferences
    - proxy preferences (note: this one still requires further updates to the content, main focus ended up being formatting)
    - label fonts
    - all the map view layers (bar the experimental one which currently doesn't have a page)
    - all the map view overlays
    - compare graph
    - graph model
    - attributes
    - types

### Why Should This Be In Core?

Updating help pages in Core

### Benefits

Help pages are more up to date

### Possible Drawbacks

Nil

### Applicable Issues

#340

### CODE REVIEW

**Review by aldebaran30701 — APPROVED** [review]
Couldn't see anything that stuck out as not being correct to me. 
Every help page that seemed to be changed has been tested by locating it's help button and verifying that it opens the correct page.
Love the work with the analytic view help pages - formatting and content is excellent.

**Review by Delphinus8821 — APPROVED** [review]

File: CorePreferences/src/au/gov/asd/tac/constellation/preferences/docs/developer-preferences.html [hunk 1]
```diff
@@ -29,12 +29,31 @@ <h2>Memory Usage</h2>
...
         <h2>Master Reset Constellation</h2>
         <p>
-            
+            Constellation is built on top of Apache Netbeans with a number of module suites and modules within those module suites.
+            Running this option is a distruptive change that will uninstall all Constellation modules, stripping the application
+            down to just the NetBeans runtime libraries.
+        </p>
+        <p>
+            Note that this option does not work for the standalone version of Constellation and it rather applicable to the version
+            of Constellation that is continously updating via the update site mechanism. This version is called the "Constellation Shell".
```
Comment (Auriga2) [inline CorePreferences/src/au/gov/asd/tac/constellation/preferences/docs/developer-preferences.html]: Typo "continously"

**Comment by Auriga2** [comment]
A help button should be added to the Schema View as well.

File: CorePreferences/src/au/gov/asd/tac/constellation/preferences/docs/developer-preferences.html (hunk 1 above)
Comment (antares1470) [reply CorePreferences/src/au/gov/asd/tac/constellation/preferences/docs/developer-preferences.html]: Good spot. Will fix

**Comment by antares1470** [comment]
> A help button should be added to the Schema View as well.

Agree that one should be added. The reason I haven't added one at this stage is that there wasn't an intuitive place to slot it in (with respect to both the gui and the code) so will probably save it for another time

**Comment by Auriga2** [comment]
Makes sense. I noticed some more views with missing help button. (Scripting view, Scatter plot, Named selection, Table View (Experimental) etc. ) 

Also the "Select All" plugin in DAV, selects only the nodes. Is it broken functionality, or should we reword the help text?

**Review by arcturus2 — APPROVED** [review]
Great stuff @antares1470

**Comment by antares1470** [comment]
> Also the "Select All" plugin in DAV, selects only the nodes. Is it broken functionality, or should we reword the help text?

@Auriga2 I don't think it is broken functionality as it appears it was written to only select all the nodes. On the other hand, I feel that select all should probably select all the transactions as well so I will go ahead and create a ticket to make that amendment.

**Pull request merged**